### PR TITLE
add ami.js library

### PR DESCRIPTION
**Pull request by NicolasRannou**
Pull request for issue: #9763 

# Profile of the lib
 * Git repository (required): https://github.com/FNNDSC/ami
 * Official website (optional, not the repository):
 * NPM package url (optional):https://www.npmjs.com/package/ami.js
 * GitHub / Bitbucket popularity (required):
   - Count of stars: 45
   - Count of watchers: 16
   - Count of forks: 16
 * NPM download stats (optional):
   - Downloads in the last day:0
   - Downloads in the last week:24
   - Downloads in the last month: 265

# Essential checklist
 * [x] I'm the author of this library
   * [x] I would like to add link to the page of this library on CDNJS on website / readme
 * [x] This lib was not found on cdnjs repo
 * [x] No already exist / duplicated issue and PR
 * [x] The lib has notable popularity
   * [ ] More than 100 [Stars / Watchers / Forks] on [GitHub / Bitbucket]
   * [ ] More than 500 downloads stats per month on npm registry
 * [x] Project has public repository on famous online hosting platform (or been hosted on npm)

# Auto-update checklist
 * [ ] Has valid tags for each versions (for git auto-update)
 * [x] Auto-update setup
 * [x] Auto-update target/source is valid.
 * [x] Auto-update filemap is correct.

# Git commit checklist
 * [ ] The first line of commit message is less then 50 chars, be clean and clear, easy to understand.
 * [ ] The parent of the commit(s) in the PR is not old than 3 days.
 * [ ] Pull request is sending from a non-master branch with meaningful name.
 * [ ] Separate unrelated changes into different commits.
 * [ ] Use rebase to squash/fixup dummy/unnecessary commits into only one commit.
 * [ ] Close corresponding issue in commit message
 * [ ] Mention related issue(s), people in commit message, comment.

### CODE REVIEW

**Comment by PeterDaveHello** [comment]
@NicolasRannou would you please also add `with npm auto-update` and `close #9763` in the commit message? Thanks.

File: ajax/libs/ami.js/package.json [hunk 1]
```diff
@@ -0,0 +1,24 @@
...
+  "npmFileMap": [
+    {
+      "basePath": "build",
+      "files": [
+        "**/*"
+      ]
+    }
+  ],
+  "filename": "ami.min.js",
+  "version": "v0.0.10",
```
Comment (PeterDaveHello) [inline ajax/libs/ami.js/package.json]: version should be `0.0.10`

**Comment by PeterDaveHello** [comment]
Looks like there is v0.0.11 on GitHub?

**Comment by NicolasRannou** [comment]
@PeterDaveHello yes but I thought only the version on NPM was relevant (currently 0.0.10 on NPM)- and hoping when pushing 0.0.11 to npm, cdnjs would just kick in and do its magics.

File: ajax/libs/ami.js/package.json (hunk 1 above)
Comment (NicolasRannou) [inline ajax/libs/ami.js/package.json]: done

Comment (NicolasRannou) [inline ajax/libs/ami.js/package.json]: was follow https://github.com/cdnjs/cdnjs/blob/master/ajax/libs/three.js/package.json which has "r" in the version

**Comment by NicolasRannou** [comment]
Closing - will re-open new one with all issues fixed and properly committed.

**Comment by PeterDaveHello** [comment]
@NicolasRannou `three.js` is a very special case, and please don't send PR via master branch, you should create another branch for the PR, thanks.

**Pull request closed without merging**